Changes

```diff
@@ -365,6 +365,9 @@ OCaml 4.07
 
 ### Manual and documentation:
 
+- PR#1779: integrate the Bigarray documentation into the main manual.
+  (Perry E. Metzger, review by Florian Angeletti and Xavier Clerc)
```

Comment:
Could you move the entry at the end of the section to keep the entries relatively sorted (the order is descirbed in CONTRIBUTING.md) and use GPR#1779 rather than PR#1779 (the convention being G for github and M for mantis) ?

Reply:
Will do. Please note the section I'm changing has LOTS of things that are labeled just "PR#" so you might look at fixing them.

Reply:
(It also has some things that are clearly out of this order. Will push a fix for my own entry in a couple of moments.)

Reply:
I do global reordering passes on a regular basis, but it's always nicer when things are ordered from boot -- it reduces the number of Changes-related conflicts, for example.

Reply:
Done!

Reply:
@gasche you might want to do a PR -> GPR pass earlier rather than later. I got the notion that PR from context. :(